unicode-data.cabal

```diff
@@ -64,19 +64,25 @@ library
   import: default-extensions, compile-options
   default-language: Haskell2010
   exposed-modules:
+      Unicode.Properties.Core
       Unicode.Properties.CombiningClass
       Unicode.Properties.Compose
       Unicode.Properties.Decompose
       Unicode.Properties.DecomposeHangul
```

Comment:
Are we going to have other modules under `Unicode` other than `Properties`? Should we consider removing it to keep the module names shorter?

Reply:
Any thoughts on `DecomposeHangul` vs `Decompose.Hangul`?

Reply:
> Are we going to have other modules under Unicode other than Properties? Should we consider removing it to keep the module names shorter?

Yes, we can remove `Properties`. It's implied anyway.

Reply:
> Any thoughts on DecomposeHangul vs Decompose.Hangul?

I think a proper thing to do is to move `decomposeCharHangul` into `Decompose` and rename `DecomposeHangul` to just `Hangul`.

Reply:
I'll do both of these in another PR.

Reply:
See #17 and #18